packages/sdk/contractkit/src/wrappers/FederatedAttestations.ts

```diff
@@ -1,8 +1,170 @@
+import { CeloTransactionObject } from '@celo/connect'
 import { FederatedAttestations } from '../generated/FederatedAttestations'
-import { BaseWrapper } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend } from './BaseWrapper'
 
-// TODO ASv2 -- add params or delete config if there are none
-// & delete other commented-out usages of this Config in CK
-// export interface FederatedAttestationsConfig {}
+type Address = string
+type Identifier = string | number[]
```

Comment:
the identifier is always a string! I feel like declaring a type variable for it isn't necessary, but up to you